dm/tests/new_relay/run.sh

```diff
@@ -287,6 +287,7 @@ function run() {
 	relay_log_count_2=$(($(ls $WORK_DIR/worker2/relay-dir/$server_uuid_2 | wc -l) - 1))
 	[ "$relay_log_count_1" -ne 1 ]
 	[ "$relay_log_count_2" -ne 1 ]
+	sleep 1
```

Comment:
why we need sleep here？

Reply:
for unstable test, I don't know the reason.

can't purge the worker2's relay log completely without `sleep 1`.

Reply:
maybe use run_dm_ctl_with_retry  to purge relay log, i am afraid of 1s is too short when load is high

Reply:
The fact is `purge-relay` returns success, but `new_relay_log_count_2>1`. So cannot use run_dm_ctl_with_retry to solve this problem.

I can go further to find the reason for it.

Reply:
this may caused by source mysql rotate it's binlog file, so maybe this sleep is no help to make test stable
pelase find the reason and fix this if you have time

Reply:
It is because `async checkpoint flush` is flushing at `mysql-bin.000003` in this moment. So it can't purge completely.

`master` branch also can triage this unstable test.

So `add sleep 1` is no problem.

Reply:
please open an issue and leave a comment around this sleep

Reply:
left in https://github.com/pingcap/tiflow/issues/4243